mysql-test/suite/galera/t/MDEV-31272.combinations

```diff
@@ -0,0 +1,4 @@
+[binlogon]
+log-bin
+
```

Comment:
Shouldn't we have also `log-slave-updates` here?

Reply:
log-slave-updates is on by default in mysql (if binary logging is enabled), but not on mariadb. now added.